locale/translations/fr_FR.json

```diff
@@ -114,6 +114,7 @@
     "page.about.author": "Auteur :",
     "page.about.license": "Licence :",
     "page.about.global_config_options": "Options de configuration globales",
+    "page.about.postgres_version": "Postgres version:",
```

Comment:
There is a space before the colon in French. The correct translation is `Version de Postgresql :`